src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -124,7 +124,7 @@ private function doParse(TokenStream $stream, ?array $names = []): Node\Node
             throw new SyntaxError(sprintf('Unexpected token "%s" of value "%s".', $stream->current->type, $stream->current->value), $stream->current->cursor, $stream->getExpression());
         }
 
-        $this->stream = null;
+        unset($this->stream);
         $this->names = null;
```

Comment:
shouldn't we unset this as well for consistency instead of using null?

Reply:
Yes.